kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -270,7 +270,11 @@ public class CodeBlock private constructor(
         if (format[p] != '%') {
           var nextP = format.nextPotentialPlaceholderPosition(startIndex = p + 1)
           if (nextP == -1) nextP = format.length
-          formatParts += format.substring(p, nextP)
+          var formatPart = format.substring(p, nextP)
```

Comment:
I think this is too generic, we should probably only fixup function bodies. I believe this code belongs somewhere in `FunSpec`, likely `emit()`.